src/Microsoft.Health.Fhir.SqlServer/Features/Search/SearchOptionsExtensions.cs

```diff
@@ -32,5 +32,24 @@ public static SortOrder GetFirstSortOrderForSupportedParam(this SearchOptions se
 
             return sortOrder;
         }
+
+        public static (Core.Models.SearchParameterInfo, SortOrder) GetFirstSupportedSortParam(this SearchOptions searchOptions)
+        {
+            EnsureArg.IsNotNull(searchOptions, nameof(searchOptions));
+
+            foreach (var sortOptions in searchOptions.Sort)
```

Comment:
When there are multiple sort fields, we need to return an error saying we only support one. Unless they specify one of the supported ones and `_lastUpdated`.